sbg/Makefile.include

```diff
@@ -14,12 +16,16 @@ SBG_ROOT := sbg
     $(SBG_ROOT)/map.cpp \
     $(SBG_ROOT)/pw_map.cpp \
     $(SBG_ROOT)/sbg.cpp \
-    $(SBG_ROOT)/sbg_algorithms.cpp
+    $(SBG_ROOT)/sbg_algorithms.cpp \
+    $(DTO_ROOT)/interval_dto.cpp \
+    $(CONVERTERS_ROOT)/interval_dto_converter.cpp
 
 include util/Makefile.include
 
 # Objects
 SBG_OBJ=$(addprefix $(BUILD_DIR)/, $(SBG_SRC:.cpp=.o))
 
 create-folders::
-	@mkdir -p $(BUILD_DIR)/$(SBG_ROOT)
+	@mkdir -p $(BUILD_DIR)/$(SBG_ROOT) \
+  @mkdir -p $(BUILD_DIR)/$(DTO_ROOT) \
```

Comment:
I'll move DTO and API related sources to a separate Makefile as part of #38

Reply:
Done in [fe8d3bb](https://github.com/CIFASIS/sb-graph/pull/37/commits/fe8d3bb877175b04087c0937b56e7ff4a218c6dd).